src/aw-client.ts

```diff
@@ -205,13 +205,6 @@ export class AWClient {
         return (await this.req.post("/0/query/", data)).data;
     }
 
-    public async import(importFile: File): Promise<any> {
-        const formData = new FormData();
```

Comment:
I assume that `FormData` is what you need from the dom lib, but you could perhaps just add this as a dependency and get it from there instead: https://github.com/form-data/form-data

Reply:
I attempted that now and realized that it was the File object/type too which is needed.

Reply:
And a File DOM object can be converted into a ReadableStream, but unfortunately the nodejs and DOM APIs for readable streams are different (ReadableStream vs stream.Readable).